spec/lib/logatron/basic_scoped_logger_spec.rb

```diff
@@ -4,8 +4,8 @@
 module Logatron
   describe BasicScopedLogger do
     it 'it supports a set of severities for logging' do
-      %w(info warn debug error critical fatal).each do |severity|
-        expect(BasicScopedLogger.method_defined? severity).to be(true)
+      Logatron::SEVERITY_MAP.keys.each do |severity|
+        expect(BasicScopedLogger.method_defined? severity.downcase).to be(true)
```

Comment:
I would prefer to see all such cases as

`Logatron::SEVERITY_MAP.keys.map(&:downcase).each do |method_name|`

Separation of concerns.